src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -139,7 +139,12 @@ function ButtonWithDropdownMenu<IValueType>({
                     onClose={() => setIsMenuVisible(false)}
                     onItemSelected={() => setIsMenuVisible(false)}
                     anchorPosition={popoverAnchorPosition}
-                    anchorRef={caretButton}
+		    anchorRef={() => {
+			if (caretButton.current && !caretButton.current.value) {
+                            caretButton.current.value = "";
```

Comment:
@jacobkim9881 can you add a null check instead of reassigning the value?

Reply:
I see. Let me replace.

Reply:
Done!